HW7.html

```diff
@@ -1,47 +1,90 @@
-         </ul>
-         <button class="redBtn">Update russophobia level</button>
-         <div class="hatredLevelBlock">
-            <label class="hatredLevelLabel">Current level:</label>
-            <span class="hatredLevelCounter"></span>
-         </div>
-      </div>
       <script>
          'use strict'
-         const _1 = document.getElementById('headerTwo')
-         const _2 = document.querySelectorAll('section h2')[0]
-         const _3 = document.querySelector('ul > li:nth-child(5)')
-         const _4 = document.getElementsByClassName('hatredLevelBlock')[0]
+
+         // 1
+         function detonatorTimer(delay) {
+            let countdown = setInterval(() => {
+               if (delay === 0) {
+                  console.log('BOOM!')
+                  clearInterval(countdown)
+               } else {
+                  console.log(delay)
+                  delay--
+               }
+            }, 1000)
+         }
+         // detonatorTimer(3)
+
+         // 2
+         function detonatorTimer(delay) {
+            setTimeout(() => {
+               let countdown = setInterval(() => {
+                  if (delay === 0) {
+                     console.log('BOOM!')
+                     clearInterval(countdown)
+                  } else {
+                     console.log(delay)
+                     delay--
+                  }
+               }, 1000)
+            }, delay * 1000)
+         }
+         // detonatorTimer(3)
+
+         // 3
+         const me = {
+            first_name: 'Serhii',
+            surname: 'Barabash',
+            nickname: 'serhash',
+            age: 28,
+            place_of_birth: 'Nikopol region',
+            region_of_living: 'Zaporizhzhia region',
+            vaccinated: true,
+            hobbies: [ 'physics', 'chemistry', 'hiking', 'singing', 'dancing', 'JS and CSS', 'philology', 'documentary', 'listening to music', 'reading', 'chess', 'interveiwing', ],
+            alma_mater: 'Nikopol metallurgical college',
+            social_media: ['Facebook', 'Telegram', 'WhatsApp'],
+            favourite_hot_drinks: [ 'cocoa', 'full fat milk', 'coffee', 'tea', 'stewed fruit', ],
+            introduce() {
+               console.log( `My name is ${this.first_name} ${this.surname}, I'm ${this.age} years old and my current region of living is ${this.region_of_living}.` )
+            },
+            plans() {
+               console.log( `I hope in the current year I'll do my best in such great spheres of my lifestyle as: ${this.hobbies.join( ', ' )} and also of course twiddlin\'.` )
+            },
+            education() {
+               console.log( `My alma mater is ${this.alma_mater}, there I'd got a qualificaion of the electrometallurgist.` )
+            },
+         }
+         // me.introduce()
+         // me.plans()
+         // me.education()
+
+         // 4
+         Object.freeze(me)
+         let securedIntroducing = () => me.introduce.bind(me),
```

Comment:
Тут можна не створювати ще одну функцію
```suggestion
         const securedIntroducing = me.introduce.bind(me);
```